website/pizzas/templates/pizzas/admin/orders.html

```diff
@@ -16,6 +16,7 @@
 
 {% block extrastyle %}
     {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="/static/admin/css/changelists.css">
```

Comment:
This extra `link` seems superfluous, since currently the stylesheet gets included twice.

Reply:
hmmm weird, I do not understand where the stylesheet comes from. But good catch!

Reply:
fixed it!